README.md

```diff
@@ -257,6 +257,7 @@ Reference
 ###### CIS Policies
 
 - [Azure Ensure SQL Server Auditing Enabled](./security/azure/sql_server_auditing/)
+- [Azure Ensure SQL Servers Have AD Admin Configured](./security/azure/sql_ad_admin/)
```

Comment:
Would maybe rename this to 'Azure Ensure SQL Server AD Admin Configured' for consistency

Reply:
Done